composer.json

```diff
@@ -26,7 +26,10 @@
         "phpunit/phpunit": "^8.3|^9.0",
         "symfony/property-access": "^4.0|^5.0",
         "symfony/serializer": "^4.0|^5.0",
-        "symfony/yaml": "^4.0|^5.0",
+        "symfony/yaml": "^4.0|^5.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.1.0"
```

Comment:
This change is incorrect. PHPUnit is a dependency of this package, not a dev dependency.

https://github.com/spatie/phpunit-snapshot-assertions/blob/808f7cf604cc9b21092b09b97f68f452a99d2d6e/src/MatchesSnapshots.php#L13

Reply:
Hi @sebastiandedeyne, I did not changed anything about the phpunit dependency. For end users, it doesn't change anything, and they should be able to install the package with phpunit 8, if they want to.

I only added a version of the require-dev version, because moreover of providing helpers for phpunit, the package use it for its own test. So the version set in require-dev will be only use in the own tests of the package.

However I understand that by doing that, we can't test anymore that the tests are passing for old versions of phpunit.

Btw it has nothing to do with the feature added here, but I've done that because the `assertFileNotExists` method was deprecated (renamed to `assertFileDoesNotExist`), and so the tests were failing in last version of phpunit.

I can revert it if you want. 🤷‍♂

Reply:
Thanks for the explanation, I didn't realise it wasn't a replacement for the dependency. Looks good to me 👍